src/components/card/_card.scss

```diff
@@ -0,0 +1,92 @@
+$cardBorderColor: $graySecondaryLight;
+$cardBorderSize: 1px;
+$cardShadow: 0 1px 3px 2px rgba(0, 0, 0, 0.05);
+
+$includeHtml: false !default;
+
+@if ($includeHtml) {
+
+  .sg-card {
+    @include component();
+    display: inline-flex;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    border-radius: $borderRadiusDefault;
+    border: $cardBorderColor solid $cardBorderSize;
+    overflow: hidden;
+    background-color: $white;
+
+    &--full {
+      width: 100%;
+
+      .sg-card__hole {
+        width: 100%;
+      }
+    }
+
+    &--with-shadow {
+      box-shadow: $cardShadow;
+    }
+
+    &--no-border {
+      border: none;
+    }
+
+    &--centered {
+
+      .sg-card__hole {
+        display: flex;
+        flex-wrap: wrap;
+        align-items: center;
+        height: 100%;
+        justify-content: center;
+      }
+    }
+
+    &--small-padding {
```

Comment:
Maybe its better to make this modifiers on holes?

Reply:
@aju @matikrk , we have the same solution for Box I think. @kdzwinel suggest that it would be better to keep modifiers on parent (I assume every hole should have the same padding, we should avoid different modifiers for holes for this component, I think)